core/src/main/java/com/novoda/noplayer/model/AudioTracks.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.noplayer.model;
+
+import com.novoda.noplayer.internal.exoplayer.mediasource.AudioTrackType;
+import com.novoda.utils.Optional;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class AudioTracks implements Iterable<PlayerAudioTrack> {
+
+    private final List<PlayerAudioTrack> audioTracks;
+
+    public static AudioTracks from(List<PlayerAudioTrack> audioTracks) {
+        return new AudioTracks(Collections.unmodifiableList(audioTracks));
+    }
+
+    private AudioTracks(List<PlayerAudioTrack> audioTracks) {
+        this.audioTracks = audioTracks;
+    }
+
+    public boolean containsPlayerAudioTrackWith(AudioTrackType trackType) {
```

Comment:
I added this **because** it made it easier and less verbose on a client project 😄 

I'll change the name of all to remove the `PlayerAudioTrack`